scripts/archive/berkeley_migration.sh

```diff
@@ -0,0 +1,675 @@
+#!/bin/bash
+
+
+# bash strict mode
+set -T # inherit DEBUG and RETURN trap for functions
+set -C # prevent file overwrite by > &> <>
+set -E # inherit -e
+set -e # exit immediately on errors
+set -u # exit on not assigned variables
+set -o pipefail # exit on pipe failure
+
+CLEAR='\033[0m'
+RED='\033[0;31m'
+
+################################################################################
+# global variable
+################################################################################
+declare -r CLI_VERSION='1.0.0';
+declare -r CLI_NAME='berkeley_migration_script.sh';
+declare -r PS4='debug($LINENO) ${FUNCNAME[0]:+${FUNCNAME[0]}}(): ';
+
+
```

Comment:
```suggestion
function check_required() {
    if ! command -v "$1" >/dev/null 2>&1; then echo "Missing required program '$1' in PATH"; exit 1; fi
}
check_required mina-berkeley-migration
check_required mina-migration-replayer
```

Reply:
Also `check_required jq` for completeness